src/less/mixins/carousel/base/carousel-mixins.less

```diff
@@ -33,6 +33,7 @@
         }
     }
 
+    &.show-control,
```

Comment:
@agliga Let's go with `&--show` (i.e. `.carousel__control--show`). I have another branch with updated docs that will demonstrate this scenario. I'll put up the PR after this one.

Reply:
Third time's the charm 👍